components/index/Volunteer.vue

```diff
@@ -45,10 +52,35 @@
         text-align: left;
       }
 
-      p {
+      > p {
```

Comment:
whats the < for?

Reply:
It means "direct child", as only direct children will be affected rather than any recursive children